src/module/Ui/templates/layout/en/serlo-home.phtml

```diff
@@ -89,75 +89,122 @@ echo $this->doctype();
                 <?php echo $this->partial('layout/partials/notifications'); ?>
 
                 <div class="flasher"></div>
+                <div class="home-header">
+                    <?php echo $this->brand()->getLogo(); ?>
+                    <?php echo $this->brand()->getBrand(); ?>
+                    <div class="header-text">The free Learning platform</div>
+                </div>
 
-                <div class="home-serlo-row home-row">
-                    <div class="home-column">
-                        <div class="logo-wrapper">
-                            <span class="serlo-logo">V</span>
-                            <div class="serlo-brand">Serlo</div>
-                        </div>
-
+                <div class="home-serlo-row">
+                    <div class="home-row">
                         <p>
-                            It is our vision to provide high quality education worldwide -
-                            completely free of charge and without ads. Like on Wikipedia,
-                            every user can create new content or improve upon existing materials.
-                            We already provide thousands of articles and solved exercises in German -
-                            now it’s time to go international.
-                        </p>
+                            It is our vision to enable personalized learning and provide high
+                            quality educational resources<br>worldwide &ndash; completely free of charge.
```

Comment:
do we need this manual line break? unrecommended because of this:

![en serlo local-4567-](https://cloud.githubusercontent.com/assets/1508477/24550655/5b3e2f78-161f-11e7-9cd5-22415ac61515.png)